seven/voice.go

```diff
@@ -25,7 +26,7 @@ type Voice struct {
 type VoiceMessage struct {
 	Error     *string `json:"error"`
 	ErrorText *string `json:"error_text"`
-	Id        *string `json:"id"`
+	Id        *int64  `json:"id"`
```

Comment:
I was unsure how large the value could be. If it is a database ID, I can imagine that it will be quite high, which is why I chose int64.

Reply:
@vorlif it is supposed to become quite high indeed, so int64 seems good to me :-)